src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,195 @@
+package duke.ui;
+
+import duke.models.Locker;
+
+import java.util.Scanner;
+import java.util.List;
+
+public class Ui {
+    private Scanner sc = new Scanner(System.in);
+
+    public String readCommand() {
+        return sc.nextLine();
+    }
+
+    /**
+     * This function responsible for printing a line.
+     */
+    public void printDash() {
+        String str = "";
+        for (int i = 0; i < 75; i++) {
+            str += "_";
+        }
+        printSpaces(str);
+    }
+
+    private void printSpaces(String printStr) {
+        System.out.println("    " + printStr);
+    }
+
+    /**
+     * This function prints the hello message every time Duke is initiated.
+     */
+    public void showWelcome() {
+        String logo = "SpongeBob";
```

Comment:
Why do you need to save `SpongeBob` as a variable if it is only used once inside the function